<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
  SPDX-FileCopyrightText: none
  SPDX-License-Identifier: GPL-3.0-or-later
 </author>
 <class>RenameTabWidget</class>
 <widget class="QWidget" name="RenameTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>119</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" columnstretch="0,1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QLineEdit" name="remoteTabTitleEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Tab title format used when a remote command (e.g. connection to another computer via SSH) is being executed</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="Konsole::TabTitleFormatButton" name="remoteTabTitleFormatButton"/>
   </item>
   <item row="0" column="2">
    <widget class="Konsole::TabTitleFormatButton" name="tabTitleFormatButton"/>
   </item>
   <item row="1" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Remote tab title format:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>remoteTabTitleEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Tab title format:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>tabTitleEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="tabTitleEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Normal tab title format</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tab Color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>tabColorCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KColorCombo" name="tabColorCombo">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>Konsole::TabTitleFormatButton</class>
   <extends>QPushButton</extends>
   <header>widgets/TabTitleFormatButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabTitleEdit</tabstop>
  <tabstop>tabTitleFormatButton</tabstop>
  <tabstop>remoteTabTitleEdit</tabstop>
  <tabstop>remoteTabTitleFormatButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
